File: AEDS/Java/Trabalho Agenda/src/Agenda/EnderecoContainer.java
package Agenda;

public class EnderecoContainer {

    public int id;
    public String logradouro;
    public int numero;
    public String complemento;
    public String bairro;
    public String cidade;
    public String estado;
    public String CEP;

    public EnderecoContainer(int id, String logradouro, int numero,
                             String complemento, String bairro, String cidade,
                             String estado, String CEP){

        this.id = id;
        this.logradouro = logradouro;
        this.numero = numero;
        this.complemento = complemento;
        this.bairro = bairro;
        this.cidade = cidade;
        this.estado = estado;
        this.CEP = CEP;

    }
}

File: AEDS/C#/Agenda em CSharp/EnderecoContainer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Agenda_em_CSharp
{
    class EnderecoContainer
    {

        public int id;
        public string logradouro;
        public int numero;
        public string complemento;
        public string bairro;
        public string cidade;
        public string estado;
        public string CEP;

        public EnderecoContainer(string Logradouro, int Numero, string Complemento, string Bairro,
                        string Cidade, string Estado, string CEP, int ID){

            this.logradouro = Logradouro;
            this.numero = Numero;
            this.complemento = Complemento;
            this.bairro = Bairro;
            this.cidade = Cidade;
            this.estado = Estado;
            this.CEP = CEP;
            this.id = ID;
        }
    }
}
